pkgs/games/steam/linux-steam-integration.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, fetchFromGitHub, gtk, meson, ninja, pkgconfig, git, pkgs, gettext, zenity }:
+
+stdenv.mkDerivation rec {
+  name = "linux-steam-integration-${version}";
+  version = "0.7.2";
+
+  nativeBuildInputs = [ meson ninja pkgconfig git gettext ];
+  buildInputs = [ gtk zenity ];
+
+  src = fetchFromGitHub {
+    owner = "solus-project";
+    repo = "linux-steam-integration";
+    rev = "851f354f9f3a87fb294dd32d68e6f932f16117f9";
+    sha256 = "0yn71fvjqi63dxk04jsndb26pgipl0nla10sy94bi7q95pk3sdf6";
+    fetchSubmodules = true;
+  };
+
+  steamBinPath = "${stdenv.lib.makeBinPath (with pkgs; [ steam ])}/steam";
+  zenityBinPath = "${stdenv.lib.makeBinPath (with pkgs; [ zenity ])}/zenity";
```

Comment:
these two can go in a let expression since they're not actual arguments for `mkDerivation`